lessons/pydata/pandas_groupby/index.ipynb

```diff
@@ -0,0 +1,3434 @@
+{
```

Comment:
<p>Bude jasné, co je <code>next(iter(...))</code>  ? Možná okomentovat, že to je stejné jako první běh <code>for</code>  cyklu?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>